module_utils/ansible_nailgun_cement.py

```diff
@@ -308,6 +319,20 @@ def find_content_view_filter(module, name, content_view, failsafe=False):
     return handle_find_response(module, content_view_filter.search(), message="No content view filter found for %s" % name, failsafe=failsafe)
 
 
+def find_host(module, name, failsafe=False):
+    host = Host(name=name).search(set(), {'search': 'name="{}"'.format(name)})
+    return handle_find_response(module, host, message="No host found for %s" % name, failsafe=failsafe)
+
+
+def find_parameter_from_hostgroup(module, hostgroup, parameter, failsafe=False):
+    return hostgroup.read_json()[parameter]
```

Comment:
Not sure, if this is better, but it is more nailgun...
```
In [30]: host_group.domain.read().fullname
Out[30]: u'deploy2.dev.atix'
```

Reply:
I wanted to create a generic function to return hostgroup-params. I am pretty sure we can reuse that later on

Reply:
I think the problem is this relies on JSON returned by Foreman as opposed to a Nailgun API. I think another option would be adding a `parameter` field to Nailgun and using that? https://github.com/SatelliteQE/nailgun/blob/master/nailgun/entities.py#L3306

Reply:
mmh ok.
probably @sean797's suggestion is the long-term correct way to do this.
for now I would go with @mdellweg's proposal to get something we can merge?
as we all now nailgun PR's take their time ;)

Reply:
I am not sure how that `parameter` field is meant. Should it not be a read_only `domain_name` field?

Reply:
I'm fine with either approach, @mdellweg would mean we wouldn't have to wait for a Nailgun PR :+1:

Reply:
ok I fixed it now, in the easiest way. I think this discussion will pop up again, once we implement more parameter that are host/host-group relevant